Makefile

```diff
@@ -159,7 +159,7 @@ docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
 # Build the manifests docker image
-docker-build-manifests: manifests ## Build docker image with the manager.
+docker-build-manifests:
```

Comment:
manifests-image?

Reply:
Other command is `build-docker` and because docker builds only images this named to `docker-build-manifests`. Does it make sense?